docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,65 @@
+# Manually Install the Calico Mesos Plugin
+This guide will walk you through manually downloading and installing the Calico-Mesos network plugin for your Mesos deployment. 
+
+The Calico-Mesos plugin must be installed with Calico's core services running on every Agent node in your cluster in order to take advantage of Calico networking. 
```

Comment:
Merge this with the statement before it by just emphasizing that this must be done once on each Agent in your cluster